capellambse/model/common/element.py

```diff
@@ -262,10 +262,16 @@ def __html__(self) -> markupsafe.Markup:
 
     def _short_html_(self) -> markupsafe.Markup:
         return markupsafe.Markup(
+            self._wrap_short_html_(
+                f" &quot;{markupsafe.Markup.escape(self.name)}&quot;"
+                f"{(': ' + str(self.value)) if hasattr(self, 'value') else ''}"
+            )
+        )
+
+    def _wrap_short_html_(self, content: str) -> str:
+        return (
```

Comment:
It would make much more sense (both semantically and from a code sharing perspective) to directly return a `markupsafe.Markup` object here instead of a `str` which then needs to be wrapped in `Markup` by the caller, seeing as this is the only use case anyway. (And it would save on a whole bunch of useless `)` lines too :) )